lib/auth/methods.go

```diff
@@ -156,7 +156,7 @@ func (a *Server) authenticateUserLogin(ctx context.Context, req AuthenticateUser
 	if err != nil {
 		event.Error = err.Error()
 		if err := a.emitter.EmitAuditEvent(a.closeCtx, event); err != nil {
-			log.WithError(err).Warn("Failed to emit login event.")
+			log.WithError(err).Warn("Failed to emit login event")
```

Comment:
Conflicts happened on log lines, of all things. All other fixes are at 240b958c09cfd57f39d48fc99c9e9b8a635df67e.